src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py

```diff
@@ -496,20 +506,27 @@ def traverse_dict(self, d, target):
         with the key being the deploy parameter and the value being the path to the value.
         e.g. {"foo": ["global", "foo", "bar"]}
 
-        param d: The dictionary to traverse.
-        param target: The regex to search for.
+        param d: The dictionary to traverse. param target: The regex to search for.
```

Comment:
Same again.  Do you need to put a colon before the param, so they don't get forced on to the same line?